.buildkite/nightly-benchmarks/nightly-pipeline.yaml

```diff
@@ -0,0 +1,294 @@
+steps: 
```

Comment:
I'm request two changes
* Use [YAML Anchor](https://joshdevlin.com/blog/yaml-repeating-sections/) to reduce the repeated `plugins kubernetes` block. 
* Add a manual [block step](https://buildkite.com/docs/pipelines/block-step) so we actually don't run them on nightly basis. Instead, they can be run manually given any commit.

Reply:
Good suggestions! Done.